lib/policies/proxy/proxy.js

```diff
@@ -83,7 +83,7 @@ module.exports = function (params, config) {
 
     logger.debug(`proxying to ${target.href}, ${req.method} ${req.url}`);
 
-    proxy.web(req, res, { target, headers });
+    proxy.web(req, res, { target, headers, buffer: req.egContext.requestStream });
```

Comment:
There's no type-checking here, and I'm not sure what error http-proxy might throw, but it would be good if it was informative actionable by the user in an Express Gateway context.  Still, going to move it forward for now.  Thanks!

Reply:
Good point. Basically, the code is `(buffer || req).pipe()` — so I guess it'd be a runtime error or something like "the object does not have `pipe` function"

Reply:
Confirmed, it's exactly the error that you'd see in case the provided object is not a `Stream`